.github/workflows/publish-images.yaml

```diff
@@ -0,0 +1,48 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    tags:
+      - "*"
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [dask-gateway, dask-gateway-server]
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
One gotcha with using the github token is that it won't trigger dependent workflows - i.e. if you want a workflow to run when you publish an image that won't happen if it was published using the `GITHUB_TOKEN`:

![image](https://user-images.githubusercontent.com/881019/155435380-d0e39945-3610-4942-85d0-6c20ecb16aba.png)

* https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow

In that case, the workaround is to use a PAT (usually stored as a repository or organisation secret). 

*Not sure if you were wanting to trigger other workflows, but if so hopefully that helps!*

Reply:
We have a bot user for this @dask-bot. There is an org secret called `DASK_BOT_TOKEN` that needs to be enabled for each repo that wants to use it, any Dask owner can do this in the org settings section. I am using it in the Dask Helm Chart because PRs raised by Actions do not trigger CI by default.

I've enabled it for this repo too so we can update this and it should be ok.